src/test/java/edu/kis/vh/nursery/DefaultCountingOutRhymerTest.java

```diff
@@ -1,11 +1,12 @@
 package edu.kis.vh.nursery;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class DefaultCountingOutRhymerTest {
     DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-    final int EMPTY_STACK_VALUE = -1;
+    final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).